botfront/imports/ui/components/stories/style.import.less

```diff
@@ -82,7 +82,63 @@
     color: rgb(33,133,208);
   }
 }
-  
+
+.branch-tab{
```

Comment:
You can safely use .stories-container { .tabular.menu { .item { ... } } } instead of .branch-tab { ... }.